### PR TITLE
Configurable frontend_ip_zones

### PR DESCRIPTION
**Pull request by r2ronoha**
## Issue number

#[97](https://github.com/Azure/terraform-azurerm-loadbalancer/issues/97)

## Checklist before requesting a review
- [X] The pr title can be used to describe what this pr did in `CHANGELOG.md` file
- [X] I have executed pre-commit on my machine
- [X] I have passed pr-check on my machine

Thanks for your cooperation!

### CODE REVIEW

**Comment by r2ronoha** [comment]
@microsoft-github-policy-service agree [company="jamf"]

**Comment by r2ronoha** [comment]
@microsoft-github-policy-service agree

**Comment by r2ronoha** [comment]
Solves https://github.com/Azure/terraform-azurerm-loadbalancer/issues/97

File: variables.tf [hunk 1]
```diff
@@ -21,6 +21,12 @@ variable "edge_zone" {
   description = "(Optional) Specifies the Edge Zone within the Azure Region where this Public IP and Load Balancer should exist. Changing this forces new resources to be created."
 }
 
+variable "frontend_ip_zones" {
+  type        = list(string)
```
Comment (lonegunmanb) [inline variables.tf:25]: Could we change the type to `set(string)` here?

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
Thanks @r2ronoha for opening this pr and apology for the late reply, could we change the new variable's type? Thanks!

File: variables.tf (hunk 1 above)
Comment (r2ronoha) [reply variables.tf]: @lonegunmanb updated

**Review by lonegunmanb — APPROVED** [review]
Thanks @r2ronoha for the update, LGTM!

**Pull request merged**